docs/modules/ROOT/pages/how-to-guides/proc_hermetic-builds.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: PROCEDURE
+
+[id="hermetic-builds_{context}"]
+= Enabling hermetic builds
+
+Hermetic builds refer to a controlled build process in which a build system knows all the required resources and dependencies. This dependency visibility ensures that the build system can accurately collect, inquire, analyze, and manage all dependencies during the build process.
+
+In {ProductName}, hermetic builds are achieved by blocking network access to the build process. This means that all the dependencies must be pre-fetched before the build can start.
+
+IMPORTANT: If you do not vendor all the required dependencies in a git repository before blocking network access, your build may fail. 
```

Comment:
```suggestion
IMPORTANT: If you do not specify all the required dependencies in a git repository before blocking network access, your build may fail. 
```

Reply:
"Vendoring" sometimes suggests that the dependencies are _copied_ into your git repo, which is not always the case, not always desired, and anyways isn't _required_ by our system.

Reply:
The point here is that if you do not have all dependencies in the git repository that enabling hermetic (not talking about pre-fetch yet) will cause the build to fail. It is more than specifying at this point. Specifying is for later with the pre-fetch functionality.

Reply:
To me this make vendoring look like the default recommendation to make hermetic builds work in RHTAP.

I don't think vendoring should be a recommendation at all, unless we're happy to rely on syft for the SBOM generation in those cases